packages/strapi-admin/admin/src/containers/MarketplacePage/index.js

```diff
@@ -38,6 +41,7 @@ const MarketPlacePage = () => {
       const response = await request('/admin/plugins/install', opts, overlayblockerParams);
 
       if (response.ok) {
+        emitEvent('didInstallPlugin', { plugin: pluginId });
```

Comment:
Installing a plugin takes quite some time I am not sure that this event will ever pop up but let's keep it.